### PR TITLE
Custom code added to primary button in theme less file

### PR DESCRIPTION
**Pull request by rajesh-mage**
## Purpose of this pull request

This pull request (PR) Created to add custom color to the theme primary button.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/css-topics/css-overview.html
-  https://devdocs.magento.com/guides/v2.4/frontend-dev-guide/css-topics/css-overview.html

### CODE REVIEW

File: src/guides/v2.3/frontend-dev-guide/css-topics/css-overview.md [hunk 1]
```diff
@@ -58,6 +58,20 @@ ExampleCorp wants to change the color of the primary buttons to orange. To achie
    @button-primary__hover__border: 1px solid @color-orange-red2;
    ```
 
+ExampleCorp wants to change the color of the primary buttons to custom color. In the Orange theme directory add the overriding app/design/frontend/ExampleCorp/orange/web/css/source/_theme.less file with the following code:
```
Comment (BarnyShergold) [inline src/guides/v2.3/frontend-dev-guide/css-topics/css-overview.md:61]:
```suggestion
ExampleCorp wants to change the color of the primary buttons to a custom color. In the Orange theme directory add the overriding `app/design/frontend/ExampleCorp/orange/web/css/source/_theme.less` file with the following code:
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Just a few small corrections

You also need to sign the Adobe CLA :)

File: src/guides/v2.3/frontend-dev-guide/css-topics/css-overview.md (hunk 1 above)
Comment (rajesh-mage) [reply src/guides/v2.3/frontend-dev-guide/css-topics/css-overview.md]: HI @BarnyShergold: 
Thanks for your review and suggestion. Requested suggestion updated.
Adobe CLA signed already and the encryption file downloaded.

Comment (dobooth) [reply src/guides/v2.3/frontend-dev-guide/css-topics/css-overview.md]: Encryption file?!

Comment (rajesh-mage) [reply src/guides/v2.3/frontend-dev-guide/css-topics/css-overview.md]: @dobooth @BarnyShergold : Encryption file?! What I meant, I have downloaded a copy of the Adobe CLA sign file.

**Comment by dobooth** [comment]
Looks like the CLA didn't take.

File: src/guides/v2.3/frontend-dev-guide/css-topics/css-overview.md [hunk 2]
```diff
@@ -58,6 +58,20 @@ ExampleCorp wants to change the color of the primary buttons to orange. To achie
    @button-primary__hover__border: 1px solid @color-orange-red2;
    ```
 
+ExampleCorp wants to change the color of the primary buttons to a custom color. In the Orange theme directory add the overriding `app/design/frontend/ExampleCorp/orange/web/css/source/_theme.less` file with the following code:
```
Comment (atwixfirster) [inline src/guides/v2.3/frontend-dev-guide/css-topics/css-overview.md:61]: Hi, all!

This sentense may confuse a reader.

There is a defference between `ExampleCorp/orange` theme and `Orange` theme. Please replace  `Orange` with `orange`.

Thank you!

Comment (rajesh-mage) [reply src/guides/v2.3/frontend-dev-guide/css-topics/css-overview.md]: Hi @atwixfirster,
Thanks for your review and suggestion. Requested suggestion updated.

Comment (atwixfirster) [reply src/guides/v2.3/frontend-dev-guide/css-topics/css-overview.md]: Hi, rajesh-mage !

Looks better for me now.

P.S. Please sign the CLA. Without that your PR will be not accepted and merged.

Thank you!

Comment (rajesh-mage) [reply src/guides/v2.3/frontend-dev-guide/css-topics/css-overview.md]: Hi, @atwixfirster
Adobe CLA sign creation is already done.

**Review by BarnyShergold — APPROVED** [review]

**Review by ajithkumar-maragathavel — APPROVED** [review]
Nice !!

**Comment by dobooth** [comment]
Hmm.....not sure why the CLA isn't taking, @rajesh-mage

**Comment by rajesh-mage** [comment]
Hi @adobe/open-source-office 
I have done the Adobe CLA sign creation process, But still, It's not reflecting on my PR.

**Comment by rajesh-mage** [comment]
@dobooth @BarnyShergold 
I have done the Adobe CLA sign creation process and downloaded the copy of the signed document, But still, It's not reflecting on my PR. Is there any other setup needed to be done?

**Comment by dobooth** [comment]
@rajesh-mage Yes, I have noticed that. I can check with someone to see what we can do.

**Comment by macdonst** [comment]
@rajesh-mage @dobooth I checked the CLA's and I see that the CLA has been signed 3 times but in each case the github user id to `https://github.com/rajesh-mage` when it needs to be `rajesh-mage`. 

Here's a screen shot from one of the agreements.

Screen Shot 2021-05-04 at 10 19 04

Please go back and sign the CLA for a fourth time but use `rajesh-mage` as the github user id. Then you close and re-open your PR to get the CLA check to run again.

**Comment by rajesh-mage** [comment]
@macdonst 
Thanks for your response. I followed the same. Now it's working fine.

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/devdocs` repository. Branch name: `imported-magento-devdocs-8901`.

**Pull request merged**